server/src/main/java/org/opensearch/action/support/master/TransportMasterNodeAction.java

```diff
@@ -201,10 +201,10 @@ protected void doStart(ClusterState clusterState) {
                         logger.debug("no known master node, scheduling a retry");
```

Comment:
line [70](https://github.com/opensearch-project/OpenSearch/pull/2519/files#diff-d20e9da696a9ce5459cbaa74987e42bea6f2389304b469ed9c05ffc20eb1be18L70) `master`->`cluster-manager`

Reply:
Good catch! Thank you, you are so carefully. Those usages are all missed... 😅